src/views/Patient/Patient.tsx

```diff
@@ -0,0 +1,57 @@
+import './Patient.css';
+import { Box, Card, CardContent, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
+
+function Patient(props: any) {
+  const client = props.client;
+
+  const [patient, setPatient] = useState(null);
```

Comment:
Type the useState with `useState<Patient | null>(null)`.   You will need to import the fhir types first.